resinhup.py

```diff
@@ -117,6 +117,13 @@ def main():
         log.error("Resinhup is not supported for the resinOS version requested.")
```

Comment:
We don't support 2.X yet, so 2. is irrelevant at present. So I say leave it as it is now. Right now the program fails if the version string is not standard (is beta).